packages/react/menu/src/Menu.tsx

```diff
@@ -66,256 +74,259 @@ const Menu = forwardRefWithAs<typeof MenuImpl, MenuOwnProps>((props, forwardedRe
       </MenuImpl>
     </Presence>
   );
-});
+}) as MenuPrimitive;
+
+type MenuImplOwnProps = Merge<
+  Polymorphic.OwnProps<typeof PopperPrimitive.Root>,
+  {
+    anchorRef: React.ComponentProps<typeof PopperPrimitive.Root>['anchorRef'];
```

Comment:
It looks like this one is actually not needed right? Would you mind double-checking?